tests/e2e/performance/csi_tests/test_pvc_multi_snapshot_performance.py

```diff
@@ -1,42 +1,445 @@
+            Delete the tested namespace
+
+        """
+        log.info("Cleanup the test environment")
+
+        # Deleting al snapshots from the cluster
+        try:
+            snapshot_list = self.snapshot.get(all_namespaces=True)["items"]
+        except Exception as err:
+            log.error(f"Cannot get the list of snapshots : {err}")
+            snapshot_list = []
+
+        log.info(f"Trying to delete all ({len(snapshot_list)}) Snapshots")
+        log.debug(
+            f"The list of all snapshots is : {json.dumps(snapshot_list, indent=3)}"
+        )
+        for vs in snapshot_list:
+            snap_name = vs["metadata"]["name"]
+            log.info(f"Try to delete {snap_name}")
+            try:
+                self.snapshot.delete(resource_name=snap_name)
+            except Exception as err:
+                log.error(f"Can not delete {snap_name} : {err}")
+
+        # Deleting the pod which wrote data to the pvc
+        log.info(f"Deleting the test POD : {self.pod_obj.name}")
+        try:
+            self.pod_obj.delete()
+        except Exception as ex:
+            log.error(f"Can not delete the test pod : {ex}")
+
+        # Deleting the PVC which used in the test.
+        log.info(f"Delete the PVC : {self.pvc_obj.name}")
+        try:
+            self.pvc_obj.delete()
+        except Exception as ex:
+            log.error(f"Can not delete the test pvc : {ex}")
+
+        # Deleting the StorageClass used in the test
+        log.info(f"Deleting the test StorageClass : {self.sc_obj.name}")
+        self.sc_obj.delete()
+
+        # Deleting the VolumeSnapshotClass used in the test
+        log.info(f"Deleting the test Snapshot Class : {self.sc_obj.name}")
+        self.snap_class.delete()
+
+        # Deleting the Data pool
+        self.delete_ceph_pool(self.sc_name)
+
+        # Deleteing the namespace used by the test
+        log.info(f"Deleting the test namespace : {self.nss_name}")
+        switch_to_default_rook_cluster_project()
+        try:
+            self.proj.delete(resource_name=self.nss_name)
+        except CommandFailed:
+            log.error(f"Can not delete project {self.nss_name}")
+            raise CommandFailed(f"{self.nss_name} was not created")
+        self.proj.wait_for_delete(resource_name=self.nss_name, timeout=60, sleep=10)
+
+        super(TestPvcMultiSnapshotPerformance, self).teardown()
+
+    def get_csi_pod(self, namespace):
+        """
+        Getting pod list in specific namespace, for the provision logs
+
+        Args:
+            namespace (str): the namespace where the pod is deployed.
+
+        Returns:
+            list : list of lines from the output of the command.
+
+        """
+        results = run_oc_command(cmd="get pod", namespace=namespace)
+        if ERRMSG in results:
+            err_msg = "Can not get the CSI controller pod"
+            log.error(err_msg)
+            raise Exception(err_msg)
+        return results
+
+    def get_log_names(self):
```

Comment:
This function too is available in https://github.com/red-hat-storage/ocs-ci/blob/master/tests/e2e/performance/csi_tests/test_multi_snapshots.py#L225, good to move it to common library.

Reply:
as above comment.